examples/multios_testing/sanity1.py

```diff
@@ -39,11 +39,11 @@ def testServerInside(self):
         self.run(" nohup nc -l %d > %s 2>&1 &" % (PORT, TFILE), shell=True, ignore_bg_processes=True)
         time.sleep(2)
         a = self.runHost("cat /etc/redhat-release").stdout.strip()
-        self.runHost("cat /etc/redhat-release | nc localhost %d" % PORT, shell=True)
+        self.runHost("Im Host | nc localhost %d" % PORT, shell=True)
```

Comment:
Can you please explain it, what do you want to test?
I don't understand at all.